pandas/core/frame.py

```diff
@@ -3642,6 +3642,10 @@ def rename(self, *args, **kwargs):
         level : int or level name, default None
             In case of a MultiIndex, only rename labels in the specified
             level.
+        tupleize_cols : boolean, default False
```

Comment:
I'd prefer not to add a new keyword argument here. I don't think that `.rename` should be able to change between Index and MultiIndex, so it shouldn't be necessary.